scripts/checkn1x_cliprompt

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 echo 'Press enter to run checkra1n in CLI mode.'
 read -s
-exec /usr/local/bin/checkra1n -cv
+exec /usr/local/bin/checkra1n --cli
```

Comment:
Correct me if I’m wrong but I’m pretty sure -c is an alias for —cli and -v for —verbose. If that’s correct (validate with —help) then revert this as the previous flags were fine.